src/main/java/Duke.java

```diff
@@ -1,23 +1,52 @@
+
 import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        Scanner myObj = new Scanner(System.in);
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
-        System.out.println("What can I do for you?");
+        System.out.println("what can i do for you");
 
-        Scanner myObj = new Scanner(System.in);
         String input = myObj.nextLine();
+        Todos[] todo = new Todos[100];
+        int c = 0;
+        while(!input.equalsIgnoreCase("bye")) {
+            if (input.equalsIgnoreCase("list")) {
+                System.out.println("-------TODO-LIST------");
+                System.out.println("----------------------");
+                for (int i = 0; i < todo.length && todo[i] != null; i++) {
+                    char x = ' ';
+                    if(todo[i].marked) {
+                        x = 'X';
+                    }
+                    System.out.println(i+1 + ". " + "[" + x + "] " + todo[i].item);
+                }
+                System.out.println("----------------------");
+            } else if (input.startsWith("mark")){
+                int i = Integer.parseInt(input.substring(5));
```

Comment:
Please consider giving a much clearer name of the variables to show its purpose instead of `int i`